modules/serverless-hotfix-patch.adoc

```diff
@@ -0,0 +1,166 @@
+
+* Patches are updates that are distributed as z-releases, for example, {ServerlessProductName} 1.29.1 is the patch that provides updates made since version 1.29.0.
+* Hotfixes are fixes that require zero downtime and are used directly in production. They are different from the usual updates in that they upgrade the customer's deployed version, and not the latest released version.
++
+Hotfixes might not be available to all customers immediately. However, changes introduced by hotfixes often become available to all customers as part of a future release.
++
+When a hotfix pertaining to your deployment is available, you will be given the hotfix CatalogSource to update your subscription and obtain the hotfix.
++
+After a new operator release is available, deployed operators with hotfixes can also be upgraded. To use the latest GA version, modify the subscription to use the public CatalogSource instead of the hotfix one.
+
+The following diagram illustrates how patches and hotfixes work:
+
+[source]
+----
+                                                                               
+     stable                                       stable-1.28                  
++--------------+                 +--------------------------------------------+
+|              |                 |                                            |
+|  +--------+  | corresponds to  |   +--------+    +--------+    +--------+   |
+|  | 1.28.0 |----------------------> | 1.28.0 |    | 1.28.1 |    | 1.28.2 |   |
+|  +--------+  |                 |   +--------+    +--------+    +--------+   |
+|              |                 |     |                   ^            |     |
+|              |                 +-----|-------------------|------------|-----+
+|  +--------+  |                created|                   |upgrades    |      
+|  | 1.28.1 |  |                from   |     hotfix_xyz    |to          |      
+|  +--------+  |                       |   +------------+  |            |      
+|              |                       +-->|            |--+            |      
+|              |                           |            |               |      
+|  +--------+  | upgrades to               +------------+               |      
+|  | 1.29.0 |<----------------------------------------------------------+      
+|  +--------+  |                                                               
+|              |                                                               
+|              |                                                               
+|  +--------+  |                                                               
+|  | 1.30.0 |  |                                                               
+|  +--------+  |                                                               
+|              |                                                               
++--------------+                                                               
+----
```

Comment:
Unfortunately, I don't think these sort of ascii diagrams are allowed in our docs. You'll have to request that they are made into formal diagrams through our team's graphic designer. You can submit a request through this form: [Technical Diagram Request Form](https://docs.google.com/forms/d/e/1FAIpQLSdF8V0n3E-aAfRen8vZchXCxwA0iDTd0QYyASnutznfGLATvA/viewform?usp=sf_link)

If you need to merge this PR before the diagrams can be made, I would recommend commenting them out pre-merge, so they're still in this file for reference. Let me know if you have any questions about this process!